Riot/Managers/PushNotification/PushNotificationManager.swift

```diff
@@ -0,0 +1,92 @@
+// 
+// Copyright 2020 Vector Creations Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import KeychainAccess
+import MatrixSDK
+
+@objcMembers
+final class PushNotificationManager: NSObject {
```

Comment:
`PushNotificationManager` only store data. It should be renamed `PushNotificationStore`. Otherwise having both `PushNotificationManager` and `PushNotificationService` can be confusing.